static/js/settings_account.js

```diff
@@ -122,6 +122,22 @@ exports.initialize_custom_date_type_fields = function (element_id) {
     $(element_id).find(".custom_user_field .datepicker").flatpickr({
         altInput: true,
         altFormat: "F j, Y"});
+
+    if (!$(element_id).find(".custom_user_field .datepicker").val()) {
+        $(element_id).find(".remove_date").hide();
+    }
```

Comment:
Remove this `if`. There is handler on this element, so this if won't be required.

Reply:
Thanks, @YJDave for reviewing my pr actually I was having a similar thought during fixing this issue but without using it gave me a web pack error.
![Screenshot from 2019-04-06 20-52-01](https://user-images.githubusercontent.com/18120004/55671471-f84ea180-58ad-11e9-823f-3d5f9380bc02.png)

Reply:
I think you have just removed `if` label, you need to remove the whole block of `if(..) {..}`.

Reply:
Oh sorry, I misunderstood you. Yes, you are right.